src/abi.js

```diff
@@ -34,17 +34,31 @@ module.exports = class ABI {
   }
 
   static oldEventSignature(event) {
-    return `${event.get('name')}(${(event.get('inputs') || [])
+    return `${event.get('name')}(${event
+      .get('inputs', [])
       .map(input => buildOldSignatureParameter(input))
```

Comment:
Can the simplified notation used in `functionSignature()` be used here as well? 
```
.map(buildSignatureParameter)
```

Reply:
Yes, done